parlai/chat_service/services/websocket/websocket_manager.py

```diff
@@ -0,0 +1,458 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""Websocket Manager Module
+Contains implementation of the WebsocketManager which helps run ParlAI via
+websockets
+"""
+
+import asyncio
+import time
+import datetime
+import logging
+import threading
+import traceback
+import sys
+from parlai.core.agents import create_agent
+
+# TODO: Use generalized module (Issue #2079)
+from parlai.chat_service.services.messenger.messenger_manager import AgentState
+
+# TODO: Use a generalized World Runner module (Issue #2079)
+from parlai.chat_service.services.messenger.world_runner import MessengerWorldRunner
+import parlai.chat_service.services.messenger.shared_utils as shared_utils
+from parlai.chat_service.services.websocket.sockets import MessageSocketHandler
+from agents import WebsocketAgent
+import tornado
+from tornado.options import options
+
+PORT = 35496
```

Comment:
just a question - is this an arbitrary number?

Reply:
Yeah arbitrary default port number that can be specified via `--port`. I found this port number in other uses of WebSockets internally

Reply:
Removed the constant from the manager and instead defined it as a default in the argument parser